API/models-json/Open-Ethics-Data-Passport.json

```diff
@@ -0,0 +1,46 @@
+{
+  "title": "Open Ethics Data Passport",
+  "type": "object",
+  "description": "OEDP Model",
+  "properties": {
+    "id": {
+      "type": "string",
+      "format": "uuid"
+    },
+    "datasets": {
+      "type": "array",
+      "items": {
+        "$ref": "./Dataset.yaml"
```

Comment:
@abhi12ravi the reference should point to the corresponding JSON file but it seems that it points to the YAML. I suggest  not to spend much time on the python fixing but directly to fix it in the JSON and their nested children JSON schemas

Reply:
@nikiluk Got it. Now, it points to `Dataset.json file`. Hope that's okay?

`        "$ref": "./Dataset.json"
`